ctapipe/__main__.py

```diff
@@ -0,0 +1,63 @@
+from traitlets.config import Application
+
+from .version import __version__
+
+
+class MainTool(Application):
+    """
+    Main entry point for ctapipe, provides other tools as subcommands
+    """
+
+    name = "ctapipe"
+    version = __version__
+
+    subcommands = {
+        "process": (
```

Comment:
I wonder if this could be built on-the fly the same way that `ctapipe info --tools` does it?  Or otherwise, maybe just drop the `--tools` option of ctapipe-info if it is no longer useful and replaced with this list. The only advantage is that it is constructed automatically.

Reply:
can reproduce, will check

Reply:
It has to do with https://github.com/cta-observatory/ctapipe/issues/656

I used the "documented" approach  in traitlets, the class method `launch_instance`. But since that calls `start`, not our custom `run`, the code does not execute correctly. Fixed by doing `InfoTool().run()`, but it should really work with `InfoTool.launch_instance()`

Reply:
Scratch that. It was the other way around. I implemented overrided "run" instead of "start"